src/services/room-service.js

```diff
@@ -74,7 +80,43 @@ export default class RoomService {
     return { room, owner, invitation };
   }
 
-  getRoomById(roomId) {
-    return this.roomStore.findOne({ _id: roomId });
+  async verifyInvitationToken({ token, user }) {
+    let roomId = null;
+    let roomName = null;
+    let isValid = false;
+
+    const invitation = await this.roomInvitationStore.findOne({ token });
+    if (invitation?.email === user.email) {
```

Comment:
I'm now wondering actually if we should persist user Id and not email with the invitation, since email will not be unique given the new Shibboleth provider 😬

Reply:
Created https://educandu.atlassian.net/browse/EDU-215